.travis.yml

```diff
@@ -20,6 +20,7 @@ branches:
   only:
     - master
 before_install:
+  - date --rfc-3339=seconds
```

Comment:
This will allow us to see in which moment in time the CI build has run, combined with the `ZONEBIE_TZ` info that's display in the build it should make it easier to reproduce possible time zone related bugs